tardis/tests/test_tardis_full.py

```diff
@@ -20,6 +20,8 @@ def data_path(fname):
 
 @pytest.mark.skipif(not pytest.config.getvalue("atomic-dataset"),
                     reason='--atomic_database was not specified')
+@pytest.mark.skipif(not pytest.config.getoption("--run-slow"),
+                    reason='this is a slow test, add --run-slow to run')
```

Comment:
**Skip if**  the flag `--run-slow` is **not** obtained by parsing the entered py.test command from command line.